tsconfig.json

```diff
@@ -26,6 +26,12 @@
       "@database/*": ["database/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    ".next/types/**/*.ts",
+    "src/app/admin/hours"
```

Comment:
???

Reply:
I dont know man. I think it happened during an npm install, I will just remove it and see what happens.